src/librustc_unicode/char.rs

```diff
@@ -234,29 +300,69 @@ impl char {
     #[inline]
     pub fn escape_default(self) -> EscapeDefault { C::escape_default(self) }
 
-    /// Returns the number of bytes this character would need if encoded in
-    /// UTF-8.
+    /// Returns the number of bytes the character this `char` would need if encoded in UTF-8.
```

Comment:
no, because the encoding really matters, it's not a well-formed statement otherwise.